conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -13,6 +13,6 @@
 # microservice specific routes
 
 
-DELETE        /test-only/journey-cache/delete-update-income             @uk.gov.hmrc.tai.controllers.JourneyCacheController.delete()
+DELETE        /tai/test-only/journey-cache/delete-update-income             @uk.gov.hmrc.tai.controllers.testOnly.TestOnlyCacheController.delete()
```

Comment:
I would use dev.routes rather than creating a new file as in dev.routes it says: 
# Any test-only end-point should be defined here